.github/workflows/simulation-1.1.yml

```diff
@@ -247,7 +247,7 @@ jobs:
     - name: Bootstrap
       run: |
         sudo apt-get --no-install-recommends install -y expect ninja-build lcov socat
-        pip install bleak
+        pip install bleak 'cryptography==43.0.0'
```

Comment:
Why would a 1.1 simulation need TCAT? I assume any 1.1 build would have TCAT disabled automatically.

Reply:
Looking at the `run` section of this job:
```cli
        ulimit -c unlimited
        ./script/test prepare_coredump_upload
        OT_OPTIONS='-DOT_TIME_SYNC=ON -DOT_FULL_LOGS=ON -DOT_LOG_OUTPUT=PLATFORM_DEFINED' VIRTUAL_TIME=0 ./script/test build expect
```
So as above show both simulation 1.1 and 1.3 for expect framework do not build 1.1 thread device, it's rather newest and runs all tests in expect folder.

Reply:
Given the fact that TCP and linkmetrics also run on simulation for 1.1 (I dont think that skip was in scope ever here). I think that the proper solution would be to create a single job for expects. To remove duplicates for 1.2 and 1.1 which look alike. If there will be decision to do so it shouldn't be a part of this PR.

@abtink @jwhui any thoughts about it?
It would be nice to sort this out since we could add certification simulation tests for TCAT.

Reply:
@EskoDijk since our experts stay silent I assume they are fine with this.